Modelica/package.mo

```diff
@@ -2278,11 +2278,11 @@ Graphical illustrations shall not be added in the diagram layer, but can be adde
 </html>"));
     end Icons;
     annotation (DocumentationClass=true,Documentation(info="<html>
-<p>A Modelica main package should be compliant with the UsersGuide stated in this documentation:</p>
+<p>User-facing packages of the Modelica Standard Library (such as <em>Modelica</em> or <em>Complex</em>) should be compliant with the UsersGuide stated in this documentation:</p>
```

Comment:
To give `Modelica` as an example doesn't seem right, as it doesn't follow the required package structure.